changelog/new_exclude_app_assets_by_default.md

```diff
@@ -0,0 +1 @@
+* [#992](https://github.com/rubocop/rubocop-rails/pull/992): Exclude `app/assets/**/*` by default. ([@r7kamura][])
```

Comment:
Can you rename this filename to `changelog/change_exclude_app_assets_by_default.md`?